Part 1

dantevibes opened the pull request.

dantevibes committed:
Added nouns and verbs 1-10

dantevibes committed:
Trying again since Github didn't take the last commit

matkatmusic commented on main.cpp:
```diff
@@ -52,86 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
+//  action 3: runs for glory
+baseballPlayer.runForGlory();
+
+//  8) 
+//  Noun: Hippy
+//  action 1: lights some incense
+hippy.lightIncense();
+//  action 2: bangs on a drum
+hippy.bangOnDrum();
+//  action 3: dances around a fire
+hippy.danceAroundFire(); 
```
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**replace** the highlighted noun/action as follows:
- [x] come up with a new noun and 3 actions
- [x] pick a noun that needs `camelCase` for the name
- [x] pick actions that need `camelCase` for their name.

matkatmusic commented on main.cpp:
```diff
@@ -52,86 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 //  9)
-//  Noun:
-//  action 1:
+//  Noun: Saxophonist
+//  action 1: plays beautiful melody
+Saxophonist.playBeautifulMelody();
+//  action 2: rips an awesome solo
+Saxophonist.ripAwesomeSolo();
+//  action 3: cleans saxophone
+Saxophonist.cleanSaxophone();
```
variable names should start with a lower-case letter.
- [x] revise

dantevibes committed:
New camelCase noun & verbs on #8. Lowercase saxophonist on #9.

dantevibes committed:
Update main.cpp

changing some arbitrary capitalization to see what happens when editing from GitHub directly.
